src/app/navbar/expandable-navbar-section/expandable-navbar-section.component.html

```diff
@@ -4,7 +4,9 @@
     (keyup.space)="isActive ? deactivateSection($event) : activateSection($event)"
     (keydown.space)="$event.preventDefault()"
     (mouseenter)="activateSection($event)"
-    (mouseleave)="deactivateSection($event)">
+    (mouseleave)="deactivateSection($event)"
+    aria-haspopup="menu"
+    aria-expanded="false">
```

Comment:
Rather than hardcoding this to `false` and changing the value in the `component.ts` code, I think you can just use `isActive` here (similar to the other attributes on this same `<div>`.  Something like this should work:
```
[attr.aria-expanded]="isActive"
```

I haven't tested it, but you shouldn't need to hardcode this.